docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1153,6 +1153,14 @@ Default: `false`
 Use of this flag can potentially result in expiration of live data. Use at your discretion.
 {{< /warning >}}
 
+## Colocation flags
```

Comment:
Maybe put flag under Sharding flags section, and link to colocation reference page.